c++/src/capnp/rpc-test.c++

```diff
@@ -1437,6 +1437,90 @@ KJ_TEST("when OutgoingRpcMessage::send() throws, we don't leak exports") {
   }
 }
 
+KJ_TEST("export the same promise twice") {
+  TestContext context;
+
+  bool exportExportIsPromise;
```

Comment:
`exportExport...`?

Reply:
o/\` I like to exportExport
I like to exportExport o/`